### ISSUE TITLE
searching and filtering papers, and simple metrics

### ISSUE THREAD

**Issue by benmarwick** [openjournals/joss#211]
This is a wonderful project, and the tooling is amazing. It may already be on your to-do list, but it would be great to be able to search or filter the papers to show only the ones of interest. 

For example, as an R monoglot, I'm keen to see all the R papers but I don't want to be distracted by papers on software for other languages and frameworks. 

It would also be handy to have some simple metrics for each paper, like number of times accessed. If the papers could sorted on these, it would be a great way to discover new software (and perhaps inspire people to explore new languages and frameworks ;)

**Comment by arfon** [comment]
Hey @benmarwick. Thanks for the feedback, it's much appreciated.

> This is a wonderful project, and the tooling is amazing. It may already be on your to-do list, but it would be great to be able to search or filter the papers to show only the ones of interest.

I totally agree. We currently don't persist much information about the actual software associated with a repository in this application (such as the programming language) but we definitely could.

> It would also be handy to have some simple metrics for each paper, like number of times accessed. If the papers could sorted on these, it would be a great way to discover new software (and perhaps inspire people to explore new languages and frameworks ;)

Again, we're not counting such things right now but I can definitely see how that information would be useful.

The main issue right now is that everyone working on JOSS is pretty focussed on editing and reviewing submissions (of which we're getting plenty!) so there's not been much time to think about enhancing the JOSS site with new features 😞 .

Which is to say, I'm happy to put these in the backlog of things we'd like to do but I currently don't have a good estimate for when I might get to this.

This application though is open source so if you're interested I'd be happy to review Pull Requests implementing some of these features but I've no idea if that's something that you'd be interested in :smile:

**Comment by benmarwick** [comment]
Thanks for your kind reply. I'm not sure that I have the skills to help much with adding tagging/searching features at the moment, but if that changes, I'll give it a go!

**Comment by kyleniemeyer** [comment]
To bump this and add some thoughts, based on discussions with a few other editors, we would like to be able to tag papers with both language but also research area/topics, and then have the ability to search or filter based on these tags.

**Comment by tracykteal** [comment]
Yes! Having this and other metadata tags will become even more important as we have more articles. It would be good to start this now. We would need to have a set of tags we use, but many people have already done metadata work, so we can look to see what's already being used and available.
